engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/serialization/ConfluentKafkaAvroDeserializationSchemaFactory.scala

```diff
@@ -34,8 +34,8 @@ trait ConfluentKafkaAvroDeserializerFactory extends LazyLogging {
   }
 }
 
-class ConfluentKafkaAvroDeserializationSchemaFactory(schemaRegistryClientFactory: ConfluentSchemaRegistryClientFactory)
-  extends KafkaAvroValueDeserializationSchemaFactory with ConfluentKafkaAvroDeserializerFactory {
+class ConfluentKafkaAvroDeserializationSchemaFactory(schemaRegistryClientFactory: ConfluentSchemaRegistryClientFactory, _useStringAsKey: Boolean)
```

Comment:
Can you name this parameter without underscore? You don't have variable shadowing here so it is not necessary

Reply:
Fixed.